src/index.html

```diff
@@ -10,13 +10,107 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <title>MOYO Header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="img"
+        href="index.html"
+      >
+        <img
+          src="images/logo.png"
+          alt="moyologo"
+        />
+      </a>
+
+      <nav class="nav">
```

Comment:
The `<nav>` element should not contain the logo. Consider moving the logo outside the `<nav>` element to maintain proper semantic structure.